examples/case_studies/multilevel_modeling.ipynb

```diff
@@ -76,7 +76,9 @@
     "from theano import tensor as tt\n",
```

Comment:
<p>Did you save the notebooks after having re-run them? The watermarks still show old "last updated" dates</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/26/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='111'/>

Reply:
<p>If there's nothing to be updated in this notebook, then perhaps it could be unstaged?</p><pre class="ql-syntax" spellcheck="false">git fetch upstream
git checkout upstream/main -- examples/case_studies/multilevel_modeling.ipynb
git add examples/case_studies/multilevel_modeling.ipynb
git commit -m 'unstage multilevel_modeling'
</pre>

Reply:
<p>Acknowledged. Thanks for the snippet!</p>